sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubConsumerClient.cs

```diff
@@ -550,9 +716,6 @@ public virtual async Task CloseAsync(CancellationToken cancellationToken = defau
         /// <remarks>
         ///   This method assumes co-ownership of the <paramref name="channel" />, marking its writer as completed
         ///   when publishing is complete or when an exception is encountered.
-        ///
-        ///   This method also assumes co-ownership of the <paramref name="publishingCancellationSource" /> and will request cancellation
```

Comment:
This was intentional; the implementation was a hold-over from before the client hierarchy refactoring when we were supporting topic/subscription semantics.  With the iterators now owning their own dedicated scope, there is no need to assume co-ownership to manage across scopes.